ms2query/ms2library.py

```diff
@@ -0,0 +1,375 @@
+from ms2query.query_from_sqlite_database import get_spectra_from_sqlite, \
+    get_tanimoto_score_for_inchikeys
+from typing import List, Dict, Any
+from matchms.Spectrum import Spectrum
+import pandas as pd
+import numpy as np
+import sqlite3
+import pickle
+from ms2deepscore.models import load_model as load_ms2ds_model
+from ms2deepscore.models import SiameseModel
+from ms2deepscore import MS2DeepScore
+from ms2deepscore.vector_operations import cosine_similarity_matrix
```

Comment:
`cosine_similarity_matrix` is imported twice (from ms2deepscore and from spec2vec), but only one should be used (they should be identical).